include/Platform/Universal Windows/UWP/RTHelpers.h

```diff
@@ -51,8 +51,8 @@
 #include <UWP/interopBase.h>
 
 #if __has_feature(objc_arc)
-#define AUTORELEASE(x)
-#define RETAIN(x)
+#define AUTORELEASE(x) x
```

Comment:
They are the "return" values here. some of these guys are supposed to return themselves so you can [[foo retain] bar ] which was creating errors somewhere. I was surprised it worked before.